src/test/java/website/frontrow/music/MusicPlayerTest.java

```diff
@@ -0,0 +1,92 @@
+package website.frontrow.music;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test the Music Player class.
+ */
+public class MusicPlayerTest
+{
+    /**
+     * Setup run before every test.
+     */
+    @Before
+    public void setup()
+    {
+        new AudioDetector();
+        if (!AudioDetector.isNoAudio())
```

Comment:
I would cancel the tests using assumeTrue (which cancels the test on false) instead.